packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.module.scss

```diff
@@ -0,0 +1,7 @@
+:local {
+  .embedded-content-container {
+    display: flex;
+    flex-direction: column;
+    height: auto;
+  }
```

Comment:
Since we are adding VisuallyHiddenText components, why does the existing CSS need to change?

Reply:
The VisuallyHiddenText was affecting the iframe resizing logic and caused some of the WDIO tests to fail due to double scrollbars being rendered where they weren't before. Adding this CSS resolved the issue.

Reply:
I am interested in how the visually hidden text was impacting the rendering.  How was the fragment being rendered in your test.  Even if a container is necessary, do we need a flexbox layout.  Should the height not be 100% of its container

Reply:
This is what it looks like normally:

![EventA_for_custom-events](https://github.com/cerner/terra-framework/assets/37230460/90b22c0d-d6c1-4d91-abe5-86032f982c8e)

This is what it looks like with the VisuallyHiddenText inside a fragment or div with no CSS changes:

![custom-events_Provider_triggers_EventA_message](https://github.com/cerner/terra-framework/assets/37230460/ae42e60e-bf59-4b4d-a199-1debc64b868c)

To remove the scrollbars, I looked into ways of getting the existing container to expand to fill the space of the parent container and landed on those flexbox changes. I am open to suggestions though and I will play around with other style changes on the existing container itself to see if we can make a change there instead to avoid adding a new container.